driver-core/src/main/com/mongodb/internal/connection/DefaultConnectionPool.java

```diff
@@ -710,7 +698,7 @@ public Prune shouldPrune(final UsageTrackingInternalConnection usageTrackingConn
     }
 
     /**
-     * Package-private methods are thread-safe,
+     * Package-access methods are thread-safe,
```

Comment:
"package-private" is not a term the JLS uses, while ["package-access"](https://docs.oracle.com/javase/specs/jls/se16/html/jls-6.html#d5e10927) is.